### PR TITLE
Text Prediction

### PR DESCRIPTION
**Pull request by ShakeelAlibhai**

### CODE REVIEW

**Comment by ApplebaumIan** [comment]
@ShakeelAlibhai Is there a tflite model or is this to just train the model?

**Comment by ShakeelAlibhai** [comment]
@ApplebaumIan It's the code to train the model, but it should save a TensorFlow model when executed.

**Comment by ApplebaumIan** [comment]
> @ApplebaumIan It's the code to train the model, but it should save a TensorFlow model when executed.

Awesome!

File: text_prediction.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+# https://www.tensorflow.org/tutorials/text/text_generation
+
+import tensorflow as tf
+
+import numpy as np
+import os
+import time
+
+path_to_file = tf.keras.utils.get_file('shakespeare.txt', 'https://storage.googleapis.com/download.tensorflow.org/data/shakespeare.txt')
```
Comment (ApplebaumIan) [inline text_prediction.py:18]: @ShakeelAlibhai in this `get_file` method. would you be able to add more than one file. Or is it one file path to train and thats it?

File: text_prediction.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+      # We pass the predicted character as the next input to the model
+      # along with the previous hidden state
+      input_eval = tf.expand_dims([predicted_id], 0)
+
+      text_generated.append(idx2char[predicted_id])
+
+  return (start_string + ''.join(text_generated))
+
+print(generate_text(model, start_string=u"ROMEO: "))
```
Comment (ApplebaumIan) [inline text_prediction.py:200]: I'm also curious as to the command to run this to see it in action!

**Comment by ShakeelAlibhai** [comment]
Pull Request Description:
* Add Python file (.py) for generating text prediction model
* Add Python notebook (.ipynb) for generating text prediction model
* Add saved model

**Pull request merged**